sequencer/src/api/endpoints.rs

```diff
@@ -21,7 +22,7 @@ use tide_disco::{
 
 #[derive(Clone, Debug, Serialize, Deserialize)]
 pub struct NamespaceProofQueryData {
-    pub proof: NamespaceProofType,
+    pub proof: crate::block::payload::NamespaceProof<Advz<Bls12_381, sha2::Sha256>>,
```

Comment:
We should import [`VidScheme`](https://hotshot.docs.espressosys.com/hotshot_types/data/type.VidScheme.html) from HotShot rather than defining `Advz` ourselves, to ensure we are using the same one.

Actually, `NamespaceProof` may not need to be generic on VID scheme at all, since we are always going to want to use the one HotShot is using. If it is generic, though, can we make a type alias that plugs in the generic parameter to make it friendlier for downstream stuff?

Reply:
can we punt this to #1047 ?

Reply:
nvm fixed in https://github.com/EspressoSystems/espresso-sequencer/pull/1087/commits/baa6c6935bb72c24249c8af25785aac026846b16